runtime/westend/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017-2021 Parity Technologies (UK) Ltd.
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright 2022 Parity Technologies (UK) Ltd.
```

Reply:
Looks like `file_header.txt` needs to be changed as well.